app/components/App.js

```diff
@@ -51,9 +72,13 @@ class App extends Component {
           currentStep={this.props.state.currentStep}
           unsavedChanges={this.props.state.unsavedChanges}
           errorCode={this.props.state.errorCode}
+          cmsStatus={this.props.state.cmsStatus}
         />
         {this._renderAppComponent()}
         <Help docName={this.props.state.helpDocument} />
+        { 'cms.isSaving' !== this.props.state.cmsStatus ? '' :
```

Comment:
I think it's more preferred to use `null` instead of a blank string to prevent any weird side-effects that might happen.